src/Conversion/FileConversionException.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace SilverStripe\Assets\Conversion;
+
+use LogicException;
+
+/**
+ * An exception that represents a failure to convert a file in a FileConverter class.
+ */
+class FileConversionException extends LogicException
```

Comment:
This allows us and developers to catch exceptions that were intentionally thrown for conversion failures - while still correctly getting errors in their faces if there are *unexpected* exceptions.